colibre/scripts/density_species_interp.py

```diff
@@ -0,0 +1,339 @@
+"""
+Makes a rho-T plot. Uses the swiftsimio library.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import glob
+
+import swiftsimio as sw
+print(sw.__file__)
+from swiftsimio import load
+import h5py as h5
+
+from unyt import mh, cm
+from matplotlib.colors import LogNorm, ListedColormap, BoundaryNorm
+from matplotlib.animation import FuncAnimation
+from matplotlib.cm import get_cmap
+from scipy.interpolate import interpn
+
+# Set the limits of the figure.
+density_bounds = [1e-3, 1e3]  # in nh/cm^3
+temperature_bounds = [10 ** (0), 10 ** (9.5)]  # in K
+dustfracs_bounds = [-4, -1]  # In metal mass fraction
+min_dustfracs = 10 ** dustfracs_bounds[0]
+bins = 64
+elements = [
+    "Hydrogen",
+    "Helium",
+    "Carbon",
+    "Nitrogen",
+    "Oxygen",
+    "Neon",
+    "Magnesium",
+    "Silicon",
+    "Sulphur",
+    "Calcium",
+    "Iron",
+    "Oatoms",]
```

Comment:
Do we not have this information from the namedcolumns instance?

Reply:
Unfortunately, there's an extra column in the cooling tables (`Oatoms`) I thought it was more explicit to write these out rather than use named columns and add one.

Reply:
I would prefer it if you used the named columns and added on "Oatoms", if at all possible.

Reply:
you also need to add Sulphur & Magnesium and remove Europium to get the same columns in an order dependent way - think it's better to write out